hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/AbstractTable.java

```diff
@@ -78,7 +78,7 @@
 
     private static final float DEFAULT_COLUMN_NAME_MIN_SIZE = 0.8F;
 
-    protected static final String ACTION_NOT_ALLOWED_MSG = "message.action.not.allowed";
+    protected static final String ACTION_NOT_ALLOWED_MSG = UIMessageIdProvider.MESSAGE_ACTION_NOT_ALLOWED;
```

Comment:
Woulnd't it be better to remove this constant and use `UIMessageIdProvider.MESSAGE_ACTION_NOT_ALLOWED` directly instead even in sup-classes?